launch/model_endpoint.py

```diff
@@ -22,11 +23,16 @@ class ModelEndpoint:
     """
 
     name: str
-    metadata: Optional[Dict] = None
+    bundle_name: Optional[str] = None
+    status: Optional[str] = None
+    destination: Optional[str] = None
+    resource_settings: Optional[dict] = None
+    worker_settings: Optional[dict] = None
+    metadata: Optional[dict] = None
     endpoint_type: Optional[str] = None
 
-    def __str__(self):
-        return f"Endpoint(name={self.name})"
+    def __repr__(self):
+        return json.dumps(self.__dict__)
```

Comment:
Curious why you need this change

Reply:
oh gonna change this - but wanted to not have to print in jupyter